pr.md

```diff
@@ -5,42 +5,18 @@ This page is really WIP
 
 ## Making a pull request to someone else's repo
 
-Open repo on github
-
-Fork with button in github top corner
-
-Clone YOUR repo not the original
-
+First navigate into the other person's repo. This is ideally done by clicking on his/her profile picture or name. This should show all the repos).
+On top right corner of the screen is a Fork button, which copies the given repo to your account.
+Finally in order to actually start working on the code it's time to download it on your computer by creating cloned repo.
+Keep in mind that the repo, you are cloning needs to be your copy of the repo (one created by Fork button) and not the original one.
+Thus the command for cloning should look like this: 
 ```
 $ git clone https://github.com/YOURusername/reponame
 ```
 
-Edit, add, commit, push
-
-Click link in `git push` output
-
-
-Clarification from chat with arrinao:
-
-```
-<Arrinao> I don't know what this is
-<Arrinao> Clone YOUR repo not the original
-<Arrinao> It's been a long time since I've done stuff on other person's repo
-<Akuli> should be:  your fork
-<Arrinao> I think only once
-<Akuli> :)(
-<Akuli> :)
-<Arrinao> with ThePhilgrim's minesweeper
-<Akuli> and in that repo, you had push permission
-<Akuli> so you just cloned it and committed and pushed, as if it was your repo
-<Arrinao> can you tell the whole sequence?
-<Arrinao> navigate to other person's repo
-<Akuli> anyway, with "YOUR repo", i meant https://github.com/Arrinao/git-guide
-<Arrinao> click fork
-<Akuli> yeah
-<Akuli> that will create https://github.com/Arrinao/git-guide
-<Akuli> (notice Arrinao)
-<Akuli> so now you have your own git-guide repo, the fork
-<Arrinao> ok
-<Akuli> and you clone this fork instead of cloning my git-guide, then you commit and push to this fork
-```
+Once this is done, you can finally start editing the code. Make all the changes you need, and when you're done with it (with preferably the final code working),
+it's time to send it back to the github - to your forked copy of the repo. This is done in three steps. First step is to add the file you've just edited. If you are
+unsure which file it was, simply navigate to your cloned repo folder in git cmd command line (cd *repofolder*) and use command 'git status', which will return all
+the info about recently changed files. Then use 'git add *edited file* command and finally git commit command. This will bring up a dialog screen, in which you're asked
+to detail the commited changes you just made. Try to write a short concise description; this will be useful in the future when browsing through lots of them. 
```

Comment:
Please refer to https://akuli.github.io/git-guide/committing.html instead of repeating everything here.